packages/terra-tabs/src/Tabs.jsx

```diff
@@ -60,34 +53,19 @@ const propTypes = {
    * Key of the pane that should be open initially.
    */
   defaultActiveKey: PropTypes.string,
-
-  /**
-   * The viewport the element will be responsive to. One of `window`, `parent` or `none`.
-   * Note: If the responsive viewport is set to `none`, then tabs never _completely_
-   * collapse into a menu.
-   */
-  responsiveTo: PropTypes.oneOf(['window', 'parent', 'none']),
```

Comment:
this should be documented as breaking change on changeLog of terra-tabs

Reply:
Updated changelog in https://github.com/cerner/terra-framework/pull/1678/commits/8b40dac988c07e97c671313470b8975ddf258da8